src/js/clinvar/eva-clinical-widget-panel.js

```diff
@@ -104,10 +104,13 @@ EvaClinicalWidgetPanel.prototype = {
 
         this.panel = Ext.create('Ext.panel.Panel', {
             header: {
-                baseCls: 'eva-header-1',
-                titlePosition: 1
+                baseCls: '',
+                titlePosition: 1,
+                style: 'padding: 5px 0px 0px 7px;'
             },
-            title:'<nav aria-label="You are here:" role="navigation"><ul class="breadcrumbs"><li><a href="?Home">EVA</a></li><li><span class="show-for-sr">Current: </span>Clinical Browser</li></ul></nav>',
+            title:'<nav aria-label="You are here:" role="navigation"><ul class="breadcrumbs"><li><a href="?Home">EVA</a></li><li><span class="show-for-sr">Current: </span>Clinical Browser</li></ul></nav><h2>Clinical Browser</h2>' +
+            '<p>Search ClinVar (release 03-2015) using any combination of the filtering options on the left hand-side</p>' +
```

Comment:
Could this text be a bit smaller, as in the wireframe? Otherwise it takes too much attention from the main table.

Also applies to the Study Browser and Variant Browser.